app/models/ability.rb

```diff
@@ -23,6 +23,8 @@ def initialize(user)
         can :close, Record
 
         can :replace_granule, Granule
+        can :delete_granule, Granule
+        can :create_granule, Collection
 
```

Comment:
So I probably also need to think about what options are available to the view, at minimum provide these options to admin, but check what options should be available for other user types (curator, daac)